pkg/boot/kexec/memory_linux.go

```diff
@@ -509,7 +508,7 @@ func (m *Memory) LoadElfSegments(r io.ReaderAt) (linux.Object, error) {
 			Start: uintptr(p.Paddr),
 			Size:  uint(p.Memsz),
 		})
-		Debug("Added segment at %#x for %#x bytes", p.Paddr, p.Memsz)
+		//Debug("Added segment at %#x for %#x bytes", p.Paddr, p.Memsz)
```

Comment:
Shouldn't we rather delete an unused line of code instead of letting it but commented out? Also I suppose the debug prints were guarded by a flag anyways?